maskpostgresdata/__init__.py

```diff
@@ -0,0 +1 @@
+from .management.commands.dump_masked_data import Command as BasePostgresDataMaskingCommand
```

Comment:
Add the trailing line here.

Maybe add `ensure_newline_at_eof_on_save` to your sublime settings.